contracts/sol6/Dao/KyberFeeHandler.sol

```diff
@@ -193,135 +196,133 @@ contract KyberFeeHandler is IKyberFeeHandler, Utils5, DaoOperator, ReentrancyGua
+            burnAmountWei
         );
-
     }
 
-    /// @dev only Dao can claim staker rewards.
+    /// @dev only kyberDao can claim staker rewards.
     /// @param staker address.
-    /// @param percentageInPrecision the relative part of the trade the staker is entitled to for this epoch.
-    ///             uint Precision: 10 ** 18 = 100%
+    /// @param percentageInPrecision the relative part of the reward the staker is entitled 
+    ///             to for this epoch.
+    ///             units Precision: 10 ** 18 = 100%
     /// @param epoch for which epoch the staker is claiming the reward
     function claimStakerReward(
         address staker,
         uint256 percentageInPrecision,
         uint256 epoch
-    ) external override onlyDAO {
+    ) external override onlyDao returns(uint256 amountWei) {
         // Amount of reward to be sent to staker
         require(percentageInPrecision <= PRECISION, "percentage too high");
-        uint256 amount = rewardsPerEpoch[epoch].mul(percentageInPrecision).div(PRECISION);
+        amountWei = rewardsPerEpoch[epoch].mul(percentageInPrecision).div(PRECISION);
 
-        // redundant check, but better revert message
-        require(totalPayoutBalance >= amount, "staker reward too high");
-        require(
-            rewardsPaidPerEpoch[epoch].add(amount) <= rewardsPerEpoch[epoch],
-            "reward paid per epoch too high"
-        );
-        rewardsPaidPerEpoch[epoch] = rewardsPaidPerEpoch[epoch].add(amount);
-        totalPayoutBalance = totalPayoutBalance.sub(amount);
+        // redundant check, can't happen
+        assert(totalPayoutBalance >= amountWei);
+        assert(rewardsPaidPerEpoch[epoch].add(amountWei) <= rewardsPerEpoch[epoch]);
+        
+        rewardsPaidPerEpoch[epoch] = rewardsPaidPerEpoch[epoch].add(amountWei);
+        totalPayoutBalance = totalPayoutBalance.sub(amountWei);
 
         // send reward to staker
-        (bool success, ) = staker.call{value: amount}("");
+        (bool success, ) = staker.call{value: amountWei}("");
         require(success, "staker rewards transfer failed");
 
-        emit RewardPaid(staker, epoch, amount);
+        emit RewardPaid(staker, epoch, amountWei);
     }
 
     /// @dev claim reabate per reserve wallet. called by any address
     /// @param rebateWallet the wallet to claim rebates for. Total accumulated rebate sent to this wallet.
-    /// @return amount of rebate claimed
+    /// @return amountWei amount of rebate claimed
     function claimReserveRebate(address rebateWallet) 
         external 
         override 
         nonReentrant 
-        returns (uint256) 
+        returns (uint256 amountWei) 
     {
         require(rebatePerWallet[rebateWallet] > 1, "no rebate to claim");
         // Get total amount of rebate accumulated
-        uint256 amount = rebatePerWallet[rebateWallet].sub(1);
+        amountWei = rebatePerWallet[rebateWallet].sub(1);
 
         // redundant check, but better revert message
```

Comment:
Maybe keep the // redundant check comments consistent
`// redundant check, can't happen`

Reply:
fixed